dep-tree-da/README.md

```diff
@@ -0,0 +1,113 @@
+Thank you for your interest in working with ActiveState as a
+developer. This assignment will help us to evaluate your skills relative
+to our working environment and is used as a primary tool in our hiring
+process. Later on in the hiring process we may have one of our developers
+pair with you on making changes or enhancements to your work.
+
+We expect this assignment to occupy no more than 3 hours of your time.
+We are primarily looking for high-quality design and documentation
+and we recognize that you may not have time to fully polish your work,
+so focus on design before other niceties.
+
+We are happy to accept solutions provided in the following formats:
+
+* ERD/ERM
+* DDL
```

Comment:
Let's move this to after "Your task is to propose a database schema..." and phrase it something like "We are happy to accept solutions in the form of ERD/ERMs or DDL".